src/Models/FieldMaps/b1_afi.m

```diff
@@ -0,0 +1,127 @@
+classdef b1_afi < AbstractModel
+% b1_afi map:  Actual Flip-Angle Imaging for B1+ mapping
+%
+% Assumptions:
+%   
+%
+% Inputs:
+%   AFIData1     3D Actual Flip Imaging (AFI) data 1
+%   AFIData2     3D Actual Flip Imaging (AFI) data 2
+%  (Mask)    Binary mask to exclude non-brain voxels
+%
+% Outputs:
+%   B1map_afi    Actual/Nominal FA map
+%
+% Protocol:
+%   Sequence    [nomFA; TR1; TR2]  nominal Flip Angle [deg]; TR1 [s]; TR2 [s]
+%
+% Options:
+%   Parallelize: Parallelize the voxelwise computation
+%     Multi CPU cores (only)
+%
+%
+%
+% Example of command line usage:
+%   Model = b1_afi;  % Create class from model
+%   Model.Prot.Sequence.Mat = txt2mat('seq.txt');  % Load protocol
+%   data = struct;  % Create data structure
+%   data.AFIData1 = load_nii_data('img1.nii.gz');
+%   data.AFIData2 = load_nii_data('img2.nii.gz');
+%   FitResults = FitData(data,Model); %fit data
+%   FitResultsSave_nii(FitResults,'img1.nii.gz'); % Save in local folder: FitResults/
+%
+%   For more examples: <a href="matlab: qMRusage(b1_afi);">qMRusage(b1_afi)</a>
+%
+% Author: 
+%
+% References:
+%   Please cite the following if you use this module:
+%     Yarnykh, VL., 2007. Actual Flip-Angle Imaging in the Pulsed Steady State: A Method for Rapid Three Dimensional Mapping of the Transmitted Radiofrequency Field. Magn. Reson. Med. 57, 192?200.
+%   In addition to citing the package:
+%     Cabana J-F, Gu Y, Boudreau M, Levesque IR, Atchia Y, Sled JG, Narayanan S, Arnold DL, Pike GB, Cohen-Adad J, Duval T, Vuong M-T and Stikov N. (2016), Quantitative magnetization transfer imaging made easy with qMTLab: Software for data simulation, analysis, and visualization. Concepts Magn. Reson.. doi: 10.1002/cmr.a.21357
+
+
+    properties (Hidden=true)
+        onlineData_url = 'https://osf.io/csjgx/'; %Please check this link. I think the version is needed for it to work
```

Comment:
For this link to work when clicking the "Download example" in the GUI, it needs to have `download/` at the end. `https://osf.io/csjgx/` is just the OSF website, so it will download an HTML web page. `https://osf.io/csjgx/download/` is a link to the data.

Reply:
Resolved in b13c1182fff

Reply:
The OSF link is now working. Thanks!